Carmu/Sources/Controllers/SessionStart/SessionStartViewController.swift

```diff
@@ -35,6 +35,21 @@ final class SessionStartViewController: UIViewController {
         // TODO: - 비동기 처리로 변경해주기
         settingData()
     }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        if SceneDelegate.isCrewCreated {
+            showGuide()
+            SceneDelegate.isCrewCreated = false // 변경을 처리한 후 다시 초기화
+        }
+    }
+
+    private func showGuide() {
+        let ruleDescriptionViewController = RuleDescriptionViewController()
+        ruleDescriptionViewController.modalPresentationStyle = .overCurrentContext
+        present(ruleDescriptionViewController, animated: true)
```

Comment:
지금 animated가 true로 돼있어서 검은 뒷배경까지 같이 밑에서 올라오는데, 그냥 false로 설정하는게 더 자연스럽지 않을까 싶습니다🙃
```suggestion
        present(ruleDescriptionViewController, animated: false)
```

Reply:
아아 다음 PR에서 수정하겠슴당 ㅋㅋ